.github/actions/setup/action.yml

```diff
@@ -14,8 +14,11 @@ runs:
       id: install-cache
       with:
         path: node_modules/
-        key: ${{ runner.os }}-install-${{ hashFiles('**/yarn.lock') }}
+        key: ${{ runner.os }}-install-${{ hashFiles('**/yarn.lock', '**/package.json') }}
 
     - if: steps.install-cache.outputs.cache-hit != 'true'
-      run: yarn install --frozen-lockfile --ignore-scripts
+      run: yarn install --ignore-scripts
+      shell: bash
+
+    - run: git diff --quiet -- yarn.lock
```

Comment:
i think the intended behavior we want is to fail this step if there are any git diffs (throw an error). will that happen with this command?

Reply:
yes, i added a comment to clarify